R/process_tests_home_care.R

```diff
@@ -0,0 +1,22 @@
+#' Process Home Care tests
+#'
+#' @description This script takes the processed Home Care extract and produces
+#' a test comparison with the previous data. This is written to disk as a csv.
+#'
+#' @param data The processed data extract
+#' @param year year of extract
+#'
+#' @return a csv document containing tests for extracts
+#' @export
+#'
+process_tests_home_care <- function(data, year) {
+  old_data <- get_existing_data_for_tests(data)
+
+  comparison <- produce_test_comparison(
+    old_data = produce_source_hc_tests(old_data),
+    new_data = produce_source_hc_tests(data)
+  ) %>%
+    write_tests_xlsx(sheet_name = "HC", year)
```

Comment:
Very minor niggle, but could we make sure that all test workbook sheet names i.e `sheet_name` in `write_tests_xlsx()` are snake_case? i.e. lowercase with underscores. It will make it much easier if we ever want to read them back in if the names are styled consistently.